PSReadLine/Cmdlets.cs

```diff
@@ -128,6 +138,8 @@ public class PSConsoleReadLineOptions
 
         public const HistorySaveStyle DefaultHistorySaveStyle = HistorySaveStyle.SaveIncrementally;
 
+        public const PredictionStyle DefaultPredictionStyle = PredictionStyle.Concise;
```

Comment:
I have concerns with this default.

Reply:
I will disable the prediction by default.